### PR TITLE
Implemented Image Creator View

### PR DESCRIPTION
**Pull request by matheweon**

### CODE REVIEW

File: Guess-E.xcodeproj/xcshareddata/xcschemes/Guess-E.xcscheme
Comment (atlant04) [inline Guess-E.xcodeproj/xcshareddata/xcschemes/Guess-E.xcscheme:1]: Did you add a new scheme? If yes, do we need one?

File: Guess-E/ImageCreatorView.swift [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+                .font(.largeTitle)
+                .bold()
+                .padding()
+            
+            ZStack(alignment: .leading) {
+                RoundedRectangle(cornerRadius: 8)
+                    .stroke(Color.white, lineWidth: 5)
+                    .background(Color(red: 118/255, green: 118/255, blue: 128/255, opacity: 0.12))
+                TextField("Enter prompt...", text: $promptText, axis: .vertical)
+                    .onReceive(promptText.publisher.collect()) { _ in
```
Comment (atlant04) [inline Guess-E/ImageCreatorView.swift:32]: it looks like you are already updating the character count below, is this onReceive necessary then?

**Review by atlant04 — COMMENTED** [review]
looks good overall, left some comments

**Comment by azooz2003-bit** [comment]
@matheweon @mathewseng  all's good, but to repeat what max said, seems like you're doing the same thing with onReceive as onChange. Lmk, then I'll merge this.

File: Guess-E.xcodeproj/xcshareddata/xcschemes/Guess-E.xcscheme
Comment (matheweon) [reply Guess-E.xcodeproj/xcshareddata/xcschemes/Guess-E.xcscheme:1]: I might have accidentally created this file by messing with the build settings. You can ignore any files in the commit except for Guess-E/ImageCreatorView.swift and Guess-E.xcodeproj/project.pbxproj

File: Guess-E/ImageCreatorView.swift (hunk 1 above)
Comment (matheweon) [reply Guess-E/ImageCreatorView.swift:32]: I'll remove that and create another PR with just necessary changes

File: Guess-E.xcodeproj/xcshareddata/xcschemes/Guess-E.xcscheme
Comment (atlant04) [reply Guess-E.xcodeproj/xcshareddata/xcschemes/Guess-E.xcscheme:1]: Can you not include that file in the commit instead? I'm not sure if your change wouldn't cause problems for others

File: Guess-E/ImageCreatorView.swift (hunk 1 above)
Comment (atlant04) [reply Guess-E/ImageCreatorView.swift:32]: Can you just include the change here?

**Pull request closed without merging**